src/client/src/App.js

```diff
@@ -22,23 +24,25 @@ class App extends Component {
         }
     }
     onKeyPress = event => {
-         if (event.key === ' ') {
-              if (this.state.text === ' ') {
-                   this.setState({text: ''});
-              } else {
-                   this.handleSubmit();
-              }
-         }
+         // if (event.key === ' ') {
+         //      if (this.state.text === ' ') {
+         //           this.setState({text: ''});
+         //      } else {
+         //           this.handleSubmit();
+         //      }
+         // }
+         this.handleSubmit();
```

Comment:
May want to re-consider this spec in future? Sending a message through the socket on every letter typed may cause some issues (too much network traffic)

Reply:
okay, whats your thoughts on this? My impression was that, as Morgan and Meia described, user can see each letters being typed/deleted in real-time. also, i found that instead of using onKeyPress props we could use onKeyDown to trigger handleSubmit() even while user pressed backspace/shift/ctrl key.

Reply:
regarding network throughput, is there anyway we could make that work? how much traffic could websocket handle?

Reply:
Yeah, I like realtime if it's possible. Would probably have to load test to know. Want to write us some automated tests? 😅 

Let's keep it live for now. We can switch it back to sending every word if we run into performance issues